contracts/examples/adder/interact/src/basic_interact.rs

```diff
@@ -7,7 +7,7 @@ use basic_interact_config::Config;
 use basic_interact_state::State;
 use clap::Parser;
 
-use multiversx_sc_snippets::imports::*;
+use multiversx_sc_snippets::{imports::*, sdk::data::keystore::InsertPassword};
```

Comment:
Let's add `sdk::data::keystore::InsertPassword` to the imports, so it's easier for developers

Reply:
Done